widgets/src/components/TaskList/TaskListContainer.js

```diff
@@ -92,12 +99,22 @@ class TaskList extends React.Component {
 
     this.setState({ loading: true });
     try {
-      const res = await getTasks(connection, page, rowsPerPage, sortedColumn, sortedOrder, filter);
+      const res = await getTasks(
+        connection,
+        page + 1,
```

Comment:
it's not starting with 0? maybe these should be a comment about this in code next to implementation or if we decide the common practice is to start from 0 - manage that in the        `getTasks` itself

Reply:
the API starts with 1, so I changed the front-end to match what's expected in the API.